src/Commands/Site/CreateCommand.php

```diff
@@ -72,7 +72,19 @@ public function create($site_name, $label, $upstream_id, $options = ['org' => nu
         if ($site = $this->getSiteById($workflow->get('waiting_for_task')->site_id)) {
             $this->log()->notice('Deploying CMS...');
             $this->processWorkflow($site->deployProduct($upstream->id));
-            $this->log()->notice('Deployed CMS');
+            $this->log()->notice('Waiting for site availability...');
+            $waits = 0;
+            do {
+                sleep(5);
+                $woke = $site->getEnvironments()->get('dev')->wake();
+                // if success is empty, then the site is still waking up.
+                $waits++;
+                // If we've waited more than 25 seconds, then something is wrong.
+                if ($waits > 5) {
+                    throw new \Exception('Could not confirm that the site is working; there might be a problem.');
+                }
+            } while (($woke['success'] ?? false) !== true);
+            $this->log()->notice('Your site has been created successfully!');
```

Comment:
I've an idea for an enhancement.  `terminus dashboard:view --print` will print the URL for the site dashboard.  Could terminus do this at the end of site creation?

Reply:
And the dev site URL so they could proceed to the CMS install?  any other useful information would be neato as well.

Reply:
Create the tickets. This PR is closing.

Reply:
Extra output should be follow-on work, just so that we can copy-edit the text and behavior in advance.